README

```diff
@@ -125,6 +125,8 @@ Direct contributions:
 =-- https://github.com/mapbase-source/source-sdk-2013/pull/124 (Memory error fixes)
```

Comment:
Duplicate line

Reply:
Good catch. I meant to reference a different PR